templates/pages/search.html

```diff
@@ -138,12 +138,13 @@ <h5 class="suggestion-title">{{lang 'forms.search.suggestions.title'}}</h5>
                 </ul>
                 {{> components/common/paginator pagination.content_results reload=true}}
             </div>
-
         {{/if}}
 
         <div id="product-listing-container" {{#if forms.search.section '!=' 'product'}}class="u-hiddenVisually"{{/if}}>
             {{> components/search/product-listing}}
         </div>
+
+        {{{region name="page_content"}}}
```

Comment:
Why is this below the search results instead of above it?

Reply:
The use case is to potentially display context-aware content based on the search rather than fix content above the search. This can change as we get feedback